WBooks/Screens/BookDetail/BookDetailFullViewController.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  BookDetailFullViewController.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 06/08/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class BookDetailFullViewController: UIViewController {
+    
+    private let _view: BookDetailFullView = BookDetailFullView.loadFromNib()!
+    private let _detailHeaderView: BookDetailView = BookDetailView.loadFromNib()!
+    
+    lazy var bookDetailViewModel: BookDetailFullViewModel = {
+        return BookDetailFullViewModel()
+    }()
+    
+    var bookViewModel: BookDetails!
+    
+    convenience init(with bookViewModel: BookDetails) {
+        self.init()
+        self.bookViewModel = bookViewModel
```

Comment:
`bookDetailViewModel` should have the bookDetails object, so in this init you can get the `BookDetailFullViewModel` from the parent